### DISCUSSION TITLE
Does CoveragePlot scale by the number of cells in a category

### DISCUSSION THREAD

**Discussion by Connorr0** [stuart-lab/signac#646] [Q&A] [answered]
I have a question about CoveragePlot. Does CoveragePlot scale by the number of cells in a category?

**Comment by timoast** [comment, marked as answer]
Yes, it normalized by number of cells in the group and average sequencing depth

**Reply by Connorr0** [reply]
Cool. Also, how does it account for average sequencing depth?

I'm mostly wondering because for some samples I have a peaks and mergepeaks assay (for combining objects). Will it try scaling by nCount_peaks or nCount_defaultassaynamehere?

**Reply by timoast** [reply]
The number of counts for the assay

**Reply by Connorr0** [reply]
Thanks!

**Comment by Aggarwal-Ayush** [comment]
Hi!
I'm wondering if this is still the case because I'm seeing a prominent peak for a cluster (c19) which has very high counts of that peak but in a very few cells (~2%). The weird thing is this peak is not even a marker of this cluster (c19) but a marker of another cluster (c0) as per FindAllMarkers function.
![file_show](https://github.com/user-attachments/assets/8b95343a-5c11-41de-9f84-2a70457bdb53)

![image](https://github.com/user-attachments/assets/9e5e88ea-0ba6-407c-b0f1-8a292cfed6b7)


A few details about my data: 10x scMultiome data of ~20 samples. I aggregated the sample counts using 10x cellranger arc aggr pipeline and then loaded the data in R Studio. I then processed the RNA and ATAC modalities individually using the integration pipeline of seurat and signac and then combined the modalities into a single seurat object for a wnn analysis.
Any insights are appreciated.
Thanks!

**Reply by timoast** [reply]
if the number of cells in a group is very low, then you can see spikes of accessibility in the coverage plot like this because the counts per cell becomes quite high. It can be better to remove tracks for extremely small clusters, or set `ymax`
